src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -1127,6 +1134,33 @@ public String toString() {
     }
   }
 
+  enum WinnerElectionMode {
+    STANDARD("standard"),
+    MULTI_SEAT_ALLOW_ONLY_ONE_WINNER_PER_ROUND("multiSeatAllowOnlyOneWinnerPerRound"),
+    MULTI_SEAT_BOTTOMS_UP("multiSeatBottomsUp"),
+    MULTI_SEAT_SEQUENTIAL_WINNER_TAKE_ALL("multiSeatSequentialWinnerTakeAll"),
```

Comment:
Another raging debate: https://english.stackexchange.com/questions/24486/status-of-verb-in-winner-take-all

I'm going to stick with "take" because that's what our RCV collaborators have been saying.

Reply:
Ugh, yuck, verb not agreeing with subject. Looks from the post like "take" is antiquated and in decline. Searched our comms and looks like only you and Rob called it that, while Caleb used "winner-takes-all". 🔥

Reply:
OK, fine!